### DISCUSSION TITLE
Enable a blog graphics to users that are unbiased to color

### DISCUSSION THREAD

**Discussion by atitsh** [twilio-labs/paste#3910] [General] [answered]
Hi, 

I have this graphics for a Segment blogpost that depicts 3 possible routes to achieve an end goal. Each route is numbered and indicated by a different color. A feedback I got on this was, this doesnt include users that are unbiased to color. 

<img width="701" alt="Screenshot 2024-05-15 at 11 01 09 AM" src="https://github.com/twilio-labs/paste/assets/104787559/975784b8-0e3b-4c06-a134-c74a625654fb">

I am soliciting creative ideas on how we can depict this better:
1. Clearly show the 3 routes 
2. Accessible to all users

**Comment by roxanagomez** [comment, marked as answer]
Hi @atitsh thank you for sharing your work with us.

Here are some recommendations to improve accessibility in your design:

- Shapes: use different shapes along with colors. For example, route one can have rounded shapes, route two can use triangles, and route three can have squares. This ensures the distinction doesn't rely solely on color.
- Number Placement: Place numbers directly on top of each route to clearly associate them with the corresponding process. Specifically, put the number over every segment of the route so the user can follow along even after passing through a node.
- Contrast: Make sure there is enough contrast between the background and colors, especially for the purple color that fades towards the end. Avoid using gradients unless they maintain good contrast throughout.
- Alternative text: If the reader is supposed to derive meaning from the image, like, it's critical to the content of the blog post, then the image shouldn't have text. Please reference the "Alternative text for media content" section of the [ Inclusive design guide](https://paste.twilio.design/inclusive-design#general-considerations)

Let us know if this feedback covers your needs or if you need further assistance. Also, could you let me know who you received the feedback from?

**Comment by atitsh** [comment]
thank you @roxanagomez for your feedback. i think number placement option works best for us. Will also look into alternative text option.

**Comment by atitsh** [comment]
Resolved. Closing discussion
